web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -255,6 +259,13 @@ const EXTERNAL_SERVICES = [
     maxsize: 1e9,
     endpoint: 'http://nwbexplorer.opensourcebrain.org/nwbfile=',
   },
+
+  {
+    name: 'VTK/ITK Viewer',
+    regex: '.ngff$',
```

Comment:
```suggestion
    regex: '.ome.zarr$',
```

Reply:
Let's allow for both?
```suggestion
    regex: '.(ngff|ome.zarr)$',
```

Reply:
didn't we decide that ngff is not a valid extension?

Reply:
according to BIDS -- yeah.  But what if we do have such a folder/file which might have... anyways -- good either way to me

Reply:
@satra @yarikoptic I went with `.ome.zarr` only here - if you agree, please resolve this conversation

Reply:
Fine with me: Although dandi-cli supports both, so we theoretically can end up with .ngff/ folders. But inability to view them would only encourage to make them properly named.

Reply:
i think the cli should stop supporting `.ngff` it's something lee and i used when we got started with writing them, but now that ome folks chimed in on it as sticking with `.ome.zarr` i think we can stick with that.